c2rust-analyze/src/context.rs

```diff
@@ -84,12 +84,25 @@ pub struct AnalysisCtxt<'a, 'tcx> {
 
     pub local_decls: &'a LocalDecls<'tcx>,
     pub local_tys: IndexVec<Local, LTy<'tcx>>,
+    pub c_void_ptrs: HashSet<Place<'tcx>>,
+    /// A mapping for substituting [`Place`]s adhering to the
+    /// following pattern
+    /// ```mir
+    /// _1 = malloc(...);
+    /// _2 = _1 as *mut T;
+    /// ```
```

Comment:
```suggestion
    /// ```
    /// where `_1` is `*c_void`.
```